Source/Core/Core/NetPlayQoS.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <enet/enet.h>
```

Comment:
You can replace this include with a forward declaration of ENetPeer.

Reply:
"In this case you may want to include the header. This is forward declaring an internal name in the enet library." @lioncash

Reply:
Oh, never mind then :)

Reply:
Yeah, it's because `ENetPeer` is a type alias (in this case, since it's a C library, a typedef) of  the struct `_ENetPeer` and you can't forward declare a type alias.